Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Investigation_and_Response.yml

```diff
@@ -0,0 +1,1937 @@
+    type: title
+    task:
+      id: 1f5ed89a-caf8-46e0-86cb-3c75e3eeb9cf
+      version: -1
+      name: Known Files
+      type: title
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#none#':
+      - '94'
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 490,
+          "y": 1270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '71':
+    id: '71'
+    taskid: f88c3ec2-83a5-4f7a-8dc1-5a5d441060bc
+    type: title
+    task:
+      id: f88c3ec2-83a5-4f7a-8dc1-5a5d441060bc
+      version: -1
+      name: Unknown Files
+      type: title
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#none#':
+      - '16'
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -80,
+          "y": 1270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '78':
+    id: '78'
+    taskid: 05246faa-b5f9-4715-8271-4f2a47da62d1
+    type: playbook
+    task:
+      id: 05246faa-b5f9-4715-8271-4f2a47da62d1
+      version: -1
+      name: CrowdStrike Falcon - Get Endpoint Forensics Data
+      description: The playbook is used to extract data from the host using RTR commands. examples of supported commands are getting a list of running processes and network connections.
+      playbookName: CrowdStrike Falcon - Get Endpoint Forensics Data
+      type: playbook
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '43'
+    scriptarguments:
+      DeviceId:
+        simple: ${incident.deviceid}
+    separatecontext: false
+    loop:
+      iscommand: false
+      exitCondition: ''
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1730,
+          "y": 2590
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '82':
+    id: '82'
+    taskid: a7cdb35a-c1de-4ee4-834f-489a16a703d4
+    type: playbook
+    task:
+      id: a7cdb35a-c1de-4ee4-834f-489a16a703d4
+      version: -1
+      name: Search For Hash In Sandbox - Generic
+      description: 'The playbook searches for a given hash in the supported sandboxes. In case the hash is known the playbook will provide the detailed analysis of the sandbox report. Currently supported sandboxes are Falcon X and Wildfire. '
+      playbookName: Search For Hash In Sandbox - Generic
+      type: playbook
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '83'
+    scriptarguments:
+      FileSha256:
+        simple: ${incident.filesha256}
```

Comment:
should we add there a unique transformer?

Reply:
Done